sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2KeyspacesResource.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.restsvc.resources.schemas;
+
+import com.codahale.metrics.annotation.Timed;
+import com.datastax.oss.driver.api.querybuilder.SchemaBuilder;
+import com.datastax.oss.driver.api.querybuilder.schema.CreateKeyspaceStart;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import io.stargate.grpc.StargateBearerToken;
+import io.stargate.proto.QueryOuterClass;
+import io.stargate.proto.StargateGrpc;
+import io.stargate.sgv2.restsvc.impl.GrpcClientFactory;
+import io.stargate.sgv2.restsvc.models.RestServiceError;
+import io.stargate.sgv2.restsvc.models.Sgv2Keyspace;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.util.Collections;
+import java.util.Map;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Api(
+    produces = MediaType.APPLICATION_JSON,
+    consumes = MediaType.APPLICATION_JSON,
+    tags = {"schemas"})
+@Path("/v2/schemas/keyspaces")
+@Produces(MediaType.APPLICATION_JSON)
+@Singleton
+public class Sgv2KeyspacesResource {
+  // Singleton resource so no need to be static
+  private final Logger logger = LoggerFactory.getLogger(getClass());
+
+  private static final SchemaBuilderHelper schemaBuilder =
+      new SchemaBuilderHelper(new JsonMapper());
+
+  /** Entity used to connect to backend gRPC service. */
+  @Inject private GrpcClientFactory grpcFactory;
+
+  @Timed
```

Comment:
Why `@Timed`?

Reply:
Dropwizard annotation to get timing metrics.

Reply:
We should go away from Dropwizard metrics for the endpoints.. Even now we would have the micrometer support we built.. Plus, I don't know if this has any effect..

Reply:
Yes, simply copied. I was wondering about moving away from JAX-RS (DropWizard is a thin layer: It's Jersey's implementation of JAX-RS spec) to something else but:

1. Main focus is to test out functionality, not to change platform (as far as I understand)
2. Many/most new Java service frameworks still use/support JAX-RS annotations.

So. I am totally fine moving to something else -- just not quite yet.